elastic/kibana
# [AO] Add alertDetailsUrl to SLO rule type

Closes #158662

## Summary

This PR adds alertDetailsUrl to the SLO burn rate rule type

|Action|Result|
|---|---|
|![image](https://github.com/elastic/kibana/assets/12370520/339e5df0-bbd3-4c02-98c5-aef8ef8c9991)|![image](https://github.com/elastic/kibana/assets/12370520/647ce4a8-16e9-4884-a8f4-3e7b5f128ebf)|

#### Filtered alert
<img src="https://github.com/elastic/kibana/assets/12370520/7ac8f529-d18e-4940-a8fe-3d148fa21d82" width="800"/>

## 🧪 How to test
- Ensure that `server.publicBaseUrl` is configured in kibana.dev.yml
- Create an SLO rule
- Add an action to this rule and specify `context.alertDetailsUrl` variable there 
- After the alert is triggered, open the value of `alertDetailsUrl` in the browser
     - you should be able to see the related alert on the `Observability > Alerts` page
